src/NzbDrone.Core/ImportLists/Trakt/TraktSettingsBase.cs

```diff
@@ -69,7 +69,7 @@ public TraktSettingsBase()
             Limit = 100;
         }
 
-        public string Link => "https://api.trakt.tv";
+        public string Link => "https://trakt.tv";
```

Comment:
Are you sure about this? Their docs do not agree is why i ask

![image](https://user-images.githubusercontent.com/8321115/127590446-308ec69c-18d5-4010-bf14-f8912c125332.png)

Reply:
That'd be exactly why I closed out the sonarr pr

Reply:
He may know something about some changes that are not documented yet and can link to it so we'll let it be for now and see